api/src/main/java/org/openmrs/module/appointmentscheduling/api/db/hibernate/HibernateAppointmentDAO.java

```diff
@@ -237,6 +238,40 @@ public Integer getCountOfAppointmentsInTimeSlotByStatus(TimeSlot timeSlot,
 				.intValue();
 	}
 
+	@Override
+	public List<Appointment> getDefaultersList(int minDays, int maxDays, Provider provider, AppointmentType appointmentType, VisitType visitType) throws DAOException {
+
+		String stringQuery = "SELECT  appointment FROM Appointment AS appointment WHERE appointment.status = :status AND appointment.voided = 0" +
```

Comment:
What if status is null?

Reply:
status is always missed, that is explicitly defined since this function only returns missed appointments